triggers/triggers.go

```diff
@@ -23,8 +23,8 @@ type NotificationTrigger struct {
 }
 
 type NotificationTemplate struct {
-	notifiers.Notification `json:"-"`
```

Comment:
If the metadata of `json:"-"` is set, the yaml element under templates is not obtained. Is it intentional?

Reply:
this is a bug. nice catch!

Reply:
cherry-picked into release-0.4 branch: https://github.com/argoproj-labs/argocd-notifications/releases/tag/v0.4.1